recipes-kernel/linux/linux-yocto/docker-fix.cfg

```diff
@@ -0,0 +1,46 @@
+CONFIG_NETFILTER_XT_MATCH_IPVS=m
```

Comment:
@Asiderr I'm guessing that enabling the first two options from this file (or even one of them) is enough